From a1f578f30be5b8f2db652c43f59a8d6ed51399d3 Mon Sep 17 00:00:00 2001
From: sbwml <admin@cooluc.com>
Date: Sat, 12 Oct 2024 07:14:42 +0800
Subject: [PATCH] random: adjust the entropy pool size to 4096 bits

Signed-off-by: sbwml <admin@cooluc.com>
---
 drivers/char/random.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/char/random.c
+++ b/drivers/char/random.c
@@ -629,7 +629,7 @@ int __cold random_prepare_cpu(unsigned i
  **********************************************************************/
 
 enum {
-	POOL_BITS = BLAKE2S_HASH_SIZE * 8,
+	POOL_BITS = BLAKE2S_HASH_SIZE * 128,
 	POOL_READY_BITS = POOL_BITS, /* When crng_init->CRNG_READY */
 	POOL_EARLY_BITS = POOL_READY_BITS / 2 /* When crng_init->CRNG_EARLY */
 };
